src/components/molecules/ConfigurationCard/ConfigurationCard.tsx

```diff
@@ -56,7 +61,16 @@ const ConfigurationCard: React.FC<ConfigurationCardProps> = props => {
           {description}
         </Text>
       </StyledHeader>
-      {children ? <StyledChildren $isActionsVisible={enabled || !isEditable}>{children}</StyledChildren> : null}
+      {children ? (
+        <>
```

Comment:
When there are no notifications, there should be at least autoscroll to the error message. Otherwise, the user will not know that something went wrong.

Reply:
Ok, I will add it

Reply:
Added